openedx/core/djangoapps/ace_common/settings/common.py

```diff
@@ -22,3 +25,29 @@ def plugin_settings(settings):  # lint-amnesty, pylint: disable=missing-function
     settings.ACE_ROUTING_KEY = ACE_ROUTING_KEY
 
     settings.FEATURES['test_django_plugin'] = True
+    settings.FCM_APP_NAME = 'fcm-edx-platform'
+
+    if getattr(settings, 'FIREBASE_SETTING_UP', None) is None:
```

Comment:
This variable name means Firebase setup status? Maybe something like `FIREBASE_SETUP_STATUS` will work here?

Reply:
Yep, renamed.